packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -383,6 +396,11 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
   };
 
   private _onMouseDown = (ev: React.MouseEvent<HTMLElement>): void => {
+    if (ev.target && portalContainsElement(ev.target as HTMLElement, this._root.current ? this._root.current : undefined)) {
```

Comment:
Since you do this check in 3 places I might make a helper function that does this to keep it all in 1 place.

Reply:
I can do that

Reply:
Done. Extracted into a function called `_isDescendantThroughPortal`